src/compiler/utilities.ts

```diff
@@ -1884,8 +1884,8 @@ namespace ts {
             writeTextOfNode,
             writeLiteral,
             writeLine,
-            increaseIndent: () => indent++,
-            decreaseIndent: () => indent--,
+            increaseIndent: () => { indent++; },

```

Comment:
If that's the case, should these functions be rewritten as `increaseIndent: () => { indent++; return indent; }` to preserve the old behavior? Or does it not matter?